include/adm/parse.hpp

```diff
@@ -44,8 +46,8 @@ namespace adm {
    * @brief Parse an XML representation of the Audio Definition Model
    *
    * Convenience wrapper for files using `parseXml(std::istream&)`
-   * @param filename XML file to parse
-   * @param options Options to influence the XML parser behaviour
+   * @param filename XML file to read
```

Comment:
this is really not too important, but why is it `read` and `reader` only here and parser everywhere else?
what am I missing?

there are more occurrences below ... is the reason to make it more "symmetric" to the "writer"  within the docs?

```suggestion
   * @param filename XML file to parse
```